common/cards/default/effects/wolf.ts

```diff
@@ -18,10 +18,10 @@ class WolfEffectCard extends EffectCard {
 
 	override canAttach(game: GameModel, pos: CardPosModel) {
 		const result = super.canAttach(game, pos)
-		const {currentPlayer} = game
+		const {player} = pos
```

Comment:
This results in wolf being able to be placed on opponent's active hermit - is this intentional?

Reply:
`EffectCard.canAttach` already checks for trying to attach to opponent, this should make checks for Emerald pass even if players have different active row indexes